### DISCUSSION TITLE
tui-combo-box

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#2598] [Q&A] [answered]
Используя ваш пример DataList с `tui-combo-box` после выбора строки, значение отображается как `[object Object]`. После щелчка мышью, значение отображается верно в `tui-combo-box`. В качестве входных данных мне необходимо использовать обычный json.
Пример на [stackblitz](https://stackblitz.com/edit/taiga-tui-combo-box-onvhe7?file=src/app/app.component.html)
Подскажите, как правильно отобразить выбранное значение в `tui-combo-box`?

**Comment by waterplea** [comment, marked as answer]
В stringify инпут надо передать функцию, которая на вход будет получать ваш объект, а возвращать его строковое представление:
https://taiga-ui.dev/components/combo-box#stringify
